internal/mongosh/mongosh_test.go

```diff
@@ -0,0 +1,16 @@
+package mongosh
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestBin(t *testing.T) {
+	want := mongoshBin
+	if isWindows() {
```

Comment:
since the github actions run on both linux and windows this should be a good enough test